libs/types/src/wsTypes.ts

```diff
@@ -1,7 +1,6 @@
 export interface Room {
   id: string;
-  creator: string;
-  members: Member[];
+  members: string[];
```

Comment:
We simplified the structure stored, and only store the username at the moment. We probably have to do something smart to be able to store more relevant data, such as `ftp` later.